src/EnergyPlus/SolarShading.cc

```diff
@@ -8519,66 +8519,59 @@ namespace SolarShading {
                                     // Equivalent Layer window model has no distinction when treating windows with and
                                     // without shades (interior, inbetween and exterior shades)
 
-                                    if (Construct(ConstrNumBack).TypeIsAirBoundaryInteriorWindow) {
-                                        TransBeamWin = 1.0;
-                                        AbsBeamWinEQL = 0.0;
-                                        AbsBeamTotWin = 0.0;
-                                    }
-                                    else {
```

Comment:
This didn't belong in here. Air boundary InteriorWindow options doesn't use the equivalent layer model.

Reply:
Good cleanup in here